----------2018/11/27 18:49:52----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.Methods.RunProxy(String ip, String Name, String pwd) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\Methods.cs:行号 39
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 124
-------------------------------------

----------2018/11/27 18:49:58----------
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 AlarmCenter.DataCenter.DataCenter.RunProxy(String csIP, String Name, String PWD, ClientType CT, Boolean bLogin)
   在 GWServiceAPI.Net.LibClass.Methods.RunProxy(String ip, String Name, String pwd) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\Methods.cs:行号 37
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 124
-------------------------------------

----------2018/11/27 18:49:58----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 18:52:14----------
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 AlarmCenter.DataCenter.DataCenter.RunProxy(String csIP, String Name, String PWD, ClientType CT, Boolean bLogin)
   在 GWServiceAPI.Net.LibClass.Methods.RunProxy(String ip, String Name, String pwd) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\Methods.cs:行号 37
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 124
-------------------------------------

----------2018/11/27 18:52:29----------
System.ServiceModel.CommunicationException: 写入管道时出错: 管道正在被关闭。 (232, 0xe8)。 ---> System.IO.PipeException: 写入管道时出错: 管道正在被关闭。 (232, 0xe8)。
   在 System.ServiceModel.Channels.PipeConnection.StartSyncWrite(Byte[] buffer, Int32 offset, Int32 size, Object& holder)
   在 System.ServiceModel.Channels.PipeConnection.StartSyncWrite(Byte[] buffer, Int32 offset, Int32 size)
   在 System.ServiceModel.Channels.PipeConnection.Write(Byte[] buffer, Int32 offset, Int32 size, Boolean immediate, TimeSpan timeout, BufferManager bufferManager)
   --- 内部异常堆栈跟踪的结尾 ---

Server stack trace: 
   在 System.ServiceModel.Channels.PipeConnection.Write(Byte[] buffer, Int32 offset, Int32 size, Boolean immediate, TimeSpan timeout, BufferManager bufferManager)
   在 System.ServiceModel.Channels.BufferedConnection.WriteNow(Byte[] buffer, Int32 offset, Int32 size, TimeSpan timeout, BufferManager bufferManager)
   在 System.ServiceModel.Channels.BufferedConnection.Write(Byte[] buffer, Int32 offset, Int32 size, Boolean immediate, TimeSpan timeout, BufferManager bufferManager)
   在 System.ServiceModel.Channels.FramingDuplexSessionChannel.OnSendCore(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.TransportDuplexSessionChannel.OnSend(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.OutputChannel.Send(Message message, TimeSpan timeout)
   在 System.ServiceModel.Dispatcher.DuplexChannelBinder.Request(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 18:52:46----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWService.Net.ClassLib.UserPermissions(String userName)
-------------------------------------

----------2018/11/27 18:52:49----------
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 AlarmCenter.DataCenter.DataCenter.RunProxy(String csIP, String Name, String PWD, ClientType CT, Boolean bLogin)
   在 GWServiceAPI.Net.LibClass.Methods.RunProxy(String ip, String Name, String pwd) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\Methods.cs:行号 37
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 124
-------------------------------------

----------2018/11/27 18:52:50----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 18:52:51----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 18:53:00----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 18:53:03----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 18:53:12----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 18:53:13----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 18:54:17----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 18:54:31----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 18:54:33----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 18:55:12----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWService.Net.ClassLib.UserPermissions(String userName)
-------------------------------------

----------2018/11/27 19:02:50----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 19:09:55----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 19:10:18----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 20:15:51----------
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 AlarmCenter.DataCenter.DataCenter.RunProxy(String csIP, String Name, String PWD, ClientType CT, Boolean bLogin)
   在 GWServiceAPI.Net.LibClass.Methods.RunProxy(String ip, String Name, String pwd) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\Methods.cs:行号 37
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 124
-------------------------------------

----------2018/11/27 20:15:54----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 20:15:56----------
System.IO.DirectoryNotFoundException: 未能找到路径“C:\MsgChat\20181127,20181120”的一部分。
   在 System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   在 System.IO.FileSystemEnumerableIterator`1.CommonInit()
   在 System.IO.FileSystemEnumerableIterator`1..ctor(String path, String originalUserPath, String searchPattern, SearchOption searchOption, SearchResultHandler`1 resultHandler, Boolean checkHost)
   在 System.IO.DirectoryInfo.InternalGetDirectories(String searchPattern, SearchOption searchOption)
   在 GWService.Net.Data.Methods.GetFileAll(String filePath, String fileName, String fileType)
-------------------------------------

----------2018/11/27 20:23:31----------
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 AlarmCenter.DataCenter.DataCenter.RunProxy(String csIP, String Name, String PWD, ClientType CT, Boolean bLogin)
   在 GWServiceAPI.Net.LibClass.Methods.RunProxy(String ip, String Name, String pwd) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\Methods.cs:行号 37
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 124
-------------------------------------

----------2018/11/27 20:23:34----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 20:24:54----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 20:26:41----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.Methods.RunProxy(String ip, String Name, String pwd) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\Methods.cs:行号 39
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 124
-------------------------------------

----------2018/11/27 20:26:44----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWService.Net.ClassLib.UserPermissions(String userName)
-------------------------------------

----------2018/11/27 20:26:45----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 20:26:49----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 20:26:49----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 20:26:58----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.Methods.RunProxy(String ip, String Name, String pwd) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\Methods.cs:行号 39
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 124
-------------------------------------

----------2018/11/27 20:27:01----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 20:27:02----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 20:27:03----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 20:27:53----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWService.Net.ClassLib.UserPermissions(String userName)
-------------------------------------

----------2018/11/27 20:30:10----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWService.Net.ClassLib.UserPermissions(String userName)
-------------------------------------

----------2018/11/27 20:30:12----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 20:30:13----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 20:30:21----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 20:30:23----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 20:31:07----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWService.Net.ClassLib.UserPermissions(String userName)
-------------------------------------

----------2018/11/27 20:32:31----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 20:34:13----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 20:43:46----------
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 AlarmCenter.DataCenter.DataCenter.RunProxy(String csIP, String Name, String PWD, ClientType CT, Boolean bLogin)
   在 GWServiceAPI.Net.LibClass.Methods.RunProxy(String ip, String Name, String pwd) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\Methods.cs:行号 37
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 124
-------------------------------------

----------2018/11/27 20:43:46----------
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 AlarmCenter.DataCenter.DataCenter.RunProxy(String csIP, String Name, String PWD, ClientType CT, Boolean bLogin)
   在 GWServiceAPI.Net.LibClass.Methods.RunProxy(String ip, String Name, String pwd) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\Methods.cs:行号 37
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 124
-------------------------------------

----------2018/11/27 20:43:50----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 20:44:25----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 20:44:41----------
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 AlarmCenter.DataCenter.DataCenter.RunProxy(String csIP, String Name, String PWD, ClientType CT, Boolean bLogin)
   在 GWServiceAPI.Net.LibClass.Methods.RunProxy(String ip, String Name, String pwd) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\Methods.cs:行号 37
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 124
-------------------------------------

----------2018/11/27 20:44:43----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWService.Net.ClassLib.UserPermissions(String userName)
-------------------------------------

----------2018/11/27 20:44:44----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 20:44:45----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 20:45:30----------
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 AlarmCenter.DataCenter.DataCenter.RunProxy(String csIP, String Name, String PWD, ClientType CT, Boolean bLogin)
   在 GWServiceAPI.Net.LibClass.Methods.RunProxy(String ip, String Name, String pwd) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\Methods.cs:行号 37
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 124
-------------------------------------

----------2018/11/27 20:45:30----------
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 AlarmCenter.DataCenter.DataCenter.RunProxy(String csIP, String Name, String PWD, ClientType CT, Boolean bLogin)
   在 GWServiceAPI.Net.LibClass.Methods.RunProxy(String ip, String Name, String pwd) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\Methods.cs:行号 37
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 124
-------------------------------------

----------2018/11/27 20:45:32----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWService.Net.ClassLib.UserPermissions(String userName)
-------------------------------------

----------2018/11/27 20:45:33----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 20:47:18----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.Methods.RunProxy(String ip, String Name, String pwd) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\Methods.cs:行号 39
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 124
-------------------------------------

----------2018/11/27 20:47:22----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 20:47:29----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWService.Net.ClassLib.UserPermissions(String userName)
-------------------------------------

----------2018/11/27 20:56:55----------
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 AlarmCenter.DataCenter.DataCenter.RunProxy(String csIP, String Name, String PWD, ClientType CT, Boolean bLogin)
   在 GWServiceAPI.Net.LibClass.Methods.RunProxy(String ip, String Name, String pwd) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\Methods.cs:行号 37
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 124
-------------------------------------

----------2018/11/27 20:56:57----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWService.Net.ClassLib.UserPermissions(String userName)
-------------------------------------

----------2018/11/27 20:56:59----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/27 20:59:45----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:06:25----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:09:46----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:10:47----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:11:20----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:12:03----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:12:56----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:14:42----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:15:07----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:16:04----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:16:15----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:17:06----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:17:23----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:19:56----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:20:24----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:20:35----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:21:03----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:21:22----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:21:40----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:22:34----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:22:48----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:23:34----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:26:50----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:26:54----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:27:04----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:27:26----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:29:14----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:36:09----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 21:38:00----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 8:46:05----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 8:46:37----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 8:48:17----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 8:50:08----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 8:50:52----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 8:51:19----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 8:51:24----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 8:52:33----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 8:53:34----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 8:55:01----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 8:55:20----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 8:56:08----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 8:56:40----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 8:58:35----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 8:59:36----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:01:15----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:02:16----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:07:10----------
System.ServiceModel.CommunicationObjectAbortedException: 无法将通信对象 System.ServiceModel.Channels.ServiceChannel 用于通信，因为它已经被中止。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.ThrowIfDisposedOrNotOpen()
   在 System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 AlarmCenterWcfServiceLibrary.IAlarmCenterService.GetUserName2SF()
   在 GWServiceAPI.Net.LibClass.ServerLib.ConnectService(String user_name) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\ServerLib.cs:行号 129
-------------------------------------

----------2018/11/28 9:08:08----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWService.Net.ClassLib.UserPermissions(String userName)
-------------------------------------

----------2018/11/28 9:08:34----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:09:05----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:09:36----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:09:57----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:10:34----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:11:00----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:11:15----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:11:37----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:11:57----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:12:07----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:12:30----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:12:46----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:15:17----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:15:46----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:17:49----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:17:55----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:18:03----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:19:06----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:20:39----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 9:54:28----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 10:07:04----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 10:07:11----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 10:09:16----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 10:10:09----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 10:11:42----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 10:15:25----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 10:15:43----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 10:16:22----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 10:22:04----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 10:22:31----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 10:23:02----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 10:23:20----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 10:24:38----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 10:25:00----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/28 10:25:59----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 18:11:13----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 19:07:34----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWService.Net.ClassLib.UserPermissions(String userName)
-------------------------------------

----------2018/11/27 19:10:20----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 19:16:25----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 19:17:00----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWService.Net.ClassLib.UserPermissions(String userName)
-------------------------------------

----------2018/11/27 19:18:31----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

----------2018/11/27 19:22:58----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWService.Net.ClassLib.UserPermissions(String userName)
-------------------------------------

----------2018/11/27 19:23:21----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWService.Net.ClassLib.UserPermissions(String userName)
-------------------------------------

----------2018/11/27 19:24:41----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWService.Net.ClassLib.UserPermissions(String userName)
-------------------------------------

----------2018/11/27 19:26:04----------
System.ArgumentException: 已添加了具有相同键的项。
   在 System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   在 System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   在 AlarmCenter.DataCenter.WebUserDictionary.AddUser(String Name)
   在 GWServiceAPI.Net.LibClass.PermissionLib.UserPermission(String auth) 位置 D:\workspace\AlarmCenterWeb\GWServiceAPI.Net\LibClass\PermissionLib.cs:行号 68
-------------------------------------

